astropy/units/equivalencies.py

```diff
@@ -687,3 +687,32 @@ def plate_scale(platescale):
                          "distance/angle")
 
     return [(si.m, si.radian, lambda d: d*platescale_val, lambda rad: rad/platescale_val)]
+
+
+def littleh(cosmologyorH0=None):
```

Comment:
Maybe `little_h` is more readable?

Reply:
I see your point and took it one step further to be `littleh_as`